src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1317,6 +1324,84 @@ protected override void HandlePauseRequestAsync(IRequestResponder<PauseArguments
             responder.SetResponse(new PauseResponse());
         }
 
+        protected override void HandleGotoRequestAsync(IRequestResponder<GotoArguments> responder)
+        {
+            responder.SetError(new ProtocolException("Not implemented exception."));
```

Comment:
>"Not implemented exception. [](start = 53, length = 27)

I might suggest instead: "Set next statement is not supported by the current debugger." since this error message might actually be show to the user.